pkg/storage/list.go

```diff
@@ -0,0 +1,14 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package storage
+
+import (
+	"storj.io/storj/pkg/paths"
+)
+
+// ListItem is a single item in a listing
+type ListItem struct {
+	path paths.Path
+	meta Meta
```

Comment:
probably need an `IsPrefix bool`

Reply:
We will add such method to the Path type, so no need for a flag here.